### PR TITLE
chore: bumping husky from 4 to 7

### PR DESCRIPTION
**Pull request by tiagoheliob**
* Bumping husky version

### CODE REVIEW

**Review by simoneb — APPROVED** [review]

**Review by simoneb — CHANGES_REQUESTED** [review]
Please use the pre-commit script as in the package.json

**Comment by tiagoheliob** [comment]
@simoneb pre-commit script fixed

File: .husky/commit-msg [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npx --no-install commitlint --edit $1
```
Comment (simoneb) [inline .husky/commit-msg:4]: please for convenience always include a new line at the end of files

File: .husky/pre-commit [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npx lint-staged
```
Comment (simoneb) [inline .husky/pre-commit:4]:
```suggestion
npx --no-install lint-staged
```

File: package.json [hunk 3]
```diff
@@ -66,18 +66,12 @@
     "eslint-plugin-import": "^2.24.2",
     "eslint-plugin-react": "^7.26.1",
     "eslint-plugin-react-hooks": "^4.2.0",
-    "husky": "4",
+    "husky": "^7.0.0",
```
Comment (simoneb) [inline package.json:69]: can you please install the latest so dependabot won't open a new PR to bump it?

**Review by simoneb — CHANGES_REQUESTED** [review]

File: .husky/commit-msg (hunk 1 above)
Comment (tiagoheliob) [reply .husky/commit-msg]: weird it isn't showing up

image

**Review by simoneb — APPROVED** [review]

**Pull request merged**